pkg/cmd/pipeline/start.go

```diff
@@ -340,7 +342,7 @@ func (opt *startOptions) getInput(pipeline *v1alpha1.Pipeline) error {
 	}
 
 	params.FilterParamsByType(pipeline.Spec.Params)
-	if len(opt.Params) == 0 && !opt.Last && opt.UsePipelineRun == "" {
+	if len(opt.Params) == 0 && !opt.Last && opt.UsePipelineRun == "" && !opt.UseParamDefaults {
```

Comment:
There should probably be some kind of error message displayed if `--use-param-defaults` and `--last` or `--use-pipelinerun` are specified to notify the user that commands should not be used together. I would check for this in [args](https://github.com/tektoncd/cli/blob/79c197d18270b6284746a24978919de590488b9c/pkg/cmd/pipeline/start.go#L116).